Gemfile

```diff
@@ -59,7 +59,11 @@ group :mdl do
 end
 
 group :doc do
-  gem "sdoc", ">= 2.6.0"
+  if ENV["EDGE"]
```

Comment:
Why we need this env check? Only edge would generate documentation for this branch.